README.md

```diff
@@ -22,11 +22,11 @@ If you find this work useful, please cite our paper
 To install the package, first create an environment, cd into it, and install the DLWPBench package via
 
 ```
-conda create -n dlwpbench python=3.11 -y && conda activate dlwpbench
+conda create -n dlwpbench netcdf4 python=3.11 numba==0.60.0 -y && conda activate dlwpbench
```

Comment:
Thought it would raise conflicts further down in the environment creation if these packages were not specified upon creation. Tested it again and it worked. Thus, I have moved the `netcdf4` and `numba` packages into the `pyproject.toml` file and pushed the changes to the pull request. Agree that it's cleaner now, thanks!